adm_program/modules/userrelations/relationtypes.php

```diff
@@ -79,6 +82,18 @@
     $relRow2['urt_name_female'] = $relRow2['urt_name_female_inverse'];
     $relationtype2->setArray($relRow2);
 
+    if((bool) $relRow['urt_edit_user'])
```

Comment:
thats not part of our code guidelines
https://www.admidio.org/dokuwiki/doku.php?id=en:entwickler:programmierrichtlinien#git_requirements